test/trip_test.rb

```diff
@@ -41,5 +44,44 @@
         end.must_raise ArgumentError
       end
     end
+
+    it "raises an error if end_time is before start_time " do
+      @trip_data[:start_time] = @trip_data[:end_time] + 15 * 60 
+      expect do
+        RideShare::Trip.new(@trip_data)
+      end.must_raise ArgumentError
+
+      @trip_data[:start_time] = @trip_data[:end_time] + 1 
+      expect do
+        RideShare::Trip.new(@trip_data)
+      end.must_raise ArgumentError
+    end
+
+    it "calculates the duration of the trip corectly" do
+      expect(@trip.duration).must_be_kind_of Float
+      expect(@trip.duration).must_equal 1500.0
```

Comment:
You should avoid using `must_equal` with `Float`s because they are imprecise.  You should use `must_be_close_to` instead.

This requires that they be within `0.1`:
```suggestion
      expect(@trip.duration).must_be_close_to 1500.0, 0.1
```